src/resources/views/columns/details_row_button.blade.php

```diff
@@ -1,4 +1,68 @@
 <!-- expand/minimize button column -->
-<td class="details-control text-center cursor-pointer">
+<span class="details-control text-center cursor-pointer m-r-5">
 	<i class="fa fa-plus-square-o details-row-button cursor-pointer" data-entry-id="{{ $entry->getKey() }}"></i>
-</td>
\ No newline at end of file
+</span>
+
+<script>
+	if (typeof registerDetailsRowButtonAction != 'function') {
+		function registerDetailsRowButtonAction() {
+	        // var crudTable = $('#crudTable tbody');
+	        // Remove any previously registered event handlers from draw.dt event callback
+	        $('#crudTable tbody').off('click', 'td .details-row-button');
+
+	        // Make sure the ajaxDatatables rows also have the correct classes
+	        $('#crudTable tbody td .details-row-button').parent('td')
+	          .removeClass('details-control').addClass('details-control')
+	          .removeClass('text-center').addClass('text-center')
+	          .removeClass('cursor-pointer').addClass('cursor-pointer');
+
+	        // Add event listener for opening and closing details
+	        $('#crudTable tbody td .details-control').on('click', function (e) {
+	        	e.stopPropagation();
+
+	            var tr = $(this).closest('tr');
+	            var btn = $(this).find('.details-row-button');
+	            var row = crud.table.row( tr );
+
+	            if ( row.child.isShown() ) {
```

Comment:
If `spacing were consistent` then `it would be good` 😄 

Why the spaces after/before the braces?

Reply:
Done, thanks!